src/main/java/org/opentripplanner/framework/application/OTPFeature.java

```diff
@@ -64,6 +64,7 @@ public enum OTPFeature {
     "Enforce transfers to happen according to the _transfers.txt_ (GTFS) and Interchanges (NeTEx). Turning this _off_ will increase the routing performance a little."
   ),
   TransmodelGraphQlApi(true, true, "Enable Transmodel (NeTEx) GraphQL API."),
+  LegacyRestApi(true, true, "Enable legacy REST API. This API will be removed in the future."),
```

Comment:
Keep sort order: Move to sandbox section, then sort  on name.

Reply:
8be001d3d0d5f9fe8c234a4ae0ee95f923fc7544